### ISSUE TITLE
java.lang.OutOfMemoryError pthread_create (1040KB stack)

### ISSUE THREAD

**Issue by farukcankaya** [intercom/intercom-android#207]
## Version info
  - Intercom for Android version: 3.0.0-beta3
  - Android OS version: 6.0
  - Android Device: lg_d855

## Logs
```````
Fatal Exception: java.lang.OutOfMemoryError: pthread_create (1040KB stack) failed: Try again
       at java.lang.Thread.nativeCreate(Thread.java)
       at java.lang.Thread.start(Thread.java:1063)
       at java.util.concurrent.ThreadPoolExecutor.addWorker(ThreadPoolExecutor.java:921)
       at java.util.concurrent.ThreadPoolExecutor.ensurePrestart(ThreadPoolExecutor.java:1556)
       at java.util.concurrent.ScheduledThreadPoolExecutor.delayedExecute(ScheduledThreadPoolExecutor.java:310)
       at java.util.concurrent.ScheduledThreadPoolExecutor.scheduleAtFixedRate(ScheduledThreadPoolExecutor.java:567)
       at java.util.concurrent.Executors$DelegatedScheduledExecutorService.scheduleAtFixedRate(Executors.java:651)
       at io.intercom.android.sdk.conversation.ConversationFragment.updateTimeStampScheduler(ConversationFragment.java:992)
       at io.intercom.android.sdk.conversation.ConversationFragment.setupWithConversation(ConversationFragment.java:890)
       at io.intercom.android.sdk.conversation.ConversationFragment.conversationSuccess(ConversationFragment.java:671)
       at java.lang.reflect.Method.invoke(Method.java)
       at io.intercom.com.squareup.otto.Bus.io.intercom.com.squareup.otto.EventHandler.handleEvent(Bus.java:3089)
       at io.intercom.com.squareup.otto.Bus.dispatchQueuedEvents(Bus.java:368)
       at io.intercom.com.squareup.otto.Bus.post(Bus.java:337)
       at io.intercom.android.sdk.api.MainThreadBus.post(MainThreadBus.java:23)
       at io.intercom.android.sdk.api.CallbackHolder$6.onSuccess(CallbackHolder.java:103)
       at io.intercom.android.sdk.api.CallbackHolder$6.onSuccess(CallbackHolder.java:101)
       at io.intercom.android.sdk.api.BaseCallback.onResponse(BaseCallback.java:30)
       at io.intercom.retrofit2.ExecutorCallAdapterFactory$ExecutorCallbackCall$1$1.run(ExecutorCallAdapterFactory.java:68)
       at android.os.Handler.handleCallback(Handler.java:739)
       at android.os.Handler.dispatchMessage(Handler.java:95)
       at android.os.Looper.loop(Looper.java:148)
       at android.app.ActivityThread.main(ActivityThread.java:5527)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:730)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:620)
```````

**Comment by ronocod** [comment]
Hi @farukcankaya,

Thanks for bringing this to our attention, we'll have a fix for this in the next beta release.

**Comment by ronocod** [comment]
Fixed in `3.0.0-beta4`.
